tests/Packages/PHPJava/Extended/_Object.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace PHPJava\Tests\Packages\PHPJava\Extended\_Object;
+
+use PHPUnit\Framework\TestCase;
+use PHPJava\Packages\java\lang\CloneNotSupportedException;
+
+class DummyObject
```

Comment:
Can you make a class another file?

Reply:
Probably this can be rewritten by anonymous class, in other words, `new class`.